apisix/admin/plugins.lua

```diff
@@ -42,6 +42,11 @@ end
 
 function _M.get(name)
     local arg = get_uri_args()
+    -- If subsystem is passed inside args then it should be oneOf: http / stream.
+    local subsystem = arg["subsystem"]
+    if subsystem and subsystem~="" and subsystem~="http" and subsystem ~= "stream" then
+        return 400, {error_msg = "no plugin found for subsystem: "..subsystem}
```

Comment:
`return 400, {error_msg = "unsupported subsystem: "..subsystem}`
Is it better?

Reply:
done